apps/myaccount/src/helpers/user.ts

```diff
@@ -86,3 +86,20 @@ export const resolveUserStoreEmbeddedUsername = (username: string): string => {
 
     return username;
 };
+
+/**
+ * This function returns the username without the user store domain prefix.
+ *
+ * @param {string} userNameWithDomain - Username of the user with the userStore domain.
+ * @return {string} User name without domain.
+ */
+export const getUserNameWithoutDomain = (userNameWithDomain: string): string => {
```

Comment:
I think it would be better to add this to `@wso2is/core/helpers`, since it could be used by `Console` as well.

Reply:
Addressed in https://github.com/wso2/identity-apps/pull/1726/commits/aaab4573659add5b2c211cf8553e637725855084